src/dtos/lookup-patch-dto.ts

```diff
@@ -0,0 +1,7 @@
+export interface LookupTablePatchDTO {
+    join_column: string;
+    description_columns: string[];
+    notes_column: string[] | undefined;
+    sort_column: string | undefined;
+    hierarchy: string | undefined;
```

Comment:
Prefer `hierarchy?: string` over `hierarchy: string | undefined`

Reply:
Agreed... Let me fix that.